scripts/submodule-diff.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Check any git submodule remotes for updates, and print difference with current Arbor repo state to `diff.log`
+
+git submodule foreach 'git describe HEAD --tags' > current_state_of_git_submodules_in_arbor_repo.log | tee current_state_of_git_submodules_in_arbor_repo.log
```

Comment:
Why do we need both `| tee` and `>` ?

Reply:
To both display the output on stdout and have it in the file.

Reply:
I just tried it out locally and it didn't display the output on stdout. But `git submodule foreach 'git describe HEAD --tags' | tee current_state_of_git_submodules_in_arbor_repo.log` does what you describe.

Reply:
Whoops :)